docs/howto.md

```diff
@@ -0,0 +1,54 @@
+#### How to create configuration for a Service Account after account creation
```

Comment:
Just create a section 

### How-To Guide

This section provides general commands on how to use the CLI interface for creating, managing and configuration spark applications. 

##### Create Service Account 

```bash
....
````

If the account "demouser" already exists, this will drop